src/components/pages/creation.tsx

```diff
@@ -1,5 +1,99 @@
 function Creation(): JSX.Element {
-  return <div>Creation</div>;
+  return (
+    <div style={{ textAlign: 'left', paddingLeft: '2.5vw' }}>
+      Creation and Deletion
+      <br />
+      <br />
+      At the heart of any project is the ability to create and delete things. So
+      let’s start with making commands.
+      <br />
+      <br />
+      touch
+      <br />
+      The touch command makes a new empty file.
+      <br />
+      Run `touch emptyFile` and see how the file system diagram changes.
+      <br />
+      If you then run `cat emptyFile`, notice how nothing is printed because the
+      file has no contents!
+      <br />
+      <br />
+      mkdir
+      <br />
+      The mkdir command makes a new empty directory.
+      <br />
+      Run `mkdir newDir` and see how the file system diagram changes.
+      <br />
+      Notice how you can now change into the newly-created directory with `cd
+      newDir`!
+      <br />
+      Removing commands are quite similar (plus, the names basically tell you
+      what the command does!).
+      <br />
+      <br />
+      rm
+      <br />
+      The rm command removes an existing file.
+      <br />
+      Looks like we have three files in the current directory.
+      <br />
+      Go ahead and remove one of them with the command `rm fileName`.
+      <br />
+      Can rm be used to remove a directory? Give it a try.
+      <br />
+      Notice how the file system diagram remains unchanged because rm cannot be
+      used to remove directories.
+      <br />
+      The terminal also printed an error message: rm: DIRNAME: is a directory
+      <br />
+      <br />
+      rmdir
+      <br />
+      The rmdir command removes an existing empty directory.
+      <br />
+      Try running `rmdir nonemptyDir`.
+      <br />
+      Notice how the terminal prints the error message: rmdir: nonemptyDir:
+      Directory not empty
+      <br />
+      We could manually rm all the files in a directory one by one, but there’s
+      an easier way: the -rf options combination.
+      <br />
+      For example,
+      <br />
+      `rm -rf nonemptyDir` recursively and forcefully (i.e., does not ask for
```

Comment:
wrap code in a span
ex> < span> rm -rf nonemptyDir < /span>

Reply:
Do for all other code

Reply:
I have added them but did not commit them for the same reason as below